### DISCUSSION TITLE
Remove Armory from fednode stack

### DISCUSSION THREAD

**Discussion by jdogresorg** [CounterpartyXCP/CIPs#94] [CIPs]
`armory` is still installed when a `counterwallet` fednode instance is spun up, and `armory` takes up a TON of disk space.

The armory project stopped getting updates in 2016 and is rarely used by anyone.

https://bitcointalk.org/index.php?topic=1351792.0

I propose that we remove armory from the `fednode` stack and comment out the code in `counterwallet` that uses armory.

**Comment by davestaxcp** [comment]
After digging through the Counterparty FAQ's and noticing this old functionality with Counterwallet, I agree if it helps the active dev's at the moment with disk space. And as you said "is rarely used by anyone."

Though it may be a good idea to communicate to any active users still using it on how to safely transition out of not using Armory with Counterwallet if they are still actively using this function. 

Also it would be good to update the FAQ on Docs for Counterwallet to represent this change when it goes live to avoid confusion in the future for somebody who may be looking to use Armory with Counterwallet.

**Comment by jdogresorg** [comment]
perhaps we just leave it in fednode and just break it off into it's own install mode... so ppl can install counterwallet without the need to install armory.... this is probably the best path forward... anyone who WANTS to run armory would still be able to.

## Fednode Install Modes
| Mode           | Services Installed                   |
| -------------- | ------------------------------------ |
| `base` 		  | bitcoin, addrindexrs, counterparty   |
| `base_extbtc`  | addrindexrs, counterparty            |
| `proxy` 		  | `base` + xcp-proxy, http-addrindexrs |
| `counterblock` | `proxy` + counterblock               |
| `counterwallet` | `counterblock` + counterwallet |
| `full`          | `counterwallet` + armory       |

**Comment by adamkrellenstein** [comment]
Armory is dead... let's delete some code :)
